TestsCalculator/CalculatorPageTests.cs

```diff
@@ -75,47 +109,60 @@ public void PositiveSelectRadioBtn()
         }
 
         [Test]
-        public void PositiveSelectDate()
+        public void PositiveSelectFutureDate()
         {
             //Arrange
-            IWebElement amountField = driver.FindElement(By.Id("amount"));
-            IWebElement percentField = driver.FindElement(By.Id("percent"));
             IWebElement termField = driver.FindElement(By.Id("term"));
             IWebElement dayDropdown = driver.FindElement(By.Id("day"));
             IWebElement monthDropdown = driver.FindElement(By.Id("month"));
             IWebElement yearDropdown = driver.FindElement(By.Id("year"));
 
             //Act
-            amountField.SendKeys("1000");
-            percentField.SendKeys("10");
             termField.SendKeys("20");
-            dayDropdown.SendKeys("7");
-            monthDropdown.SendKeys("Jun");
-            yearDropdown.SendKeys("2022");
+            new SelectElement(dayDropdown).SelectByText("7");
+            new SelectElement(monthDropdown).SelectByText("June");
+            new SelectElement(yearDropdown).SelectByText("2022");
 
             //Assert
             IWebElement endDateField = driver.FindElement(By.Id("endDate"));
             Assert.AreEqual("27/06/2022", endDateField.GetAttribute("value"));
         }
+
+        [Test]
+        public void PositiveSelectPastDate()
```

Comment:
You also may merge this, 'CheckBoundaryForDays' and 'PositiveSelectFutureDate' tests.